### DISCUSSION TITLE
How to change duration format in custom Twig templates?

### DISCUSSION THREAD

**Discussion by ghost** [kimai/kimai#3497] [Questions & Answers] [answered]
Hello everyone,

currently durations are formatted like this in my rendered twig invoice template: **00:30 h**

Code I am using: `{{ entry.duration|duration(isDecimal) }}`

But I want that it looks like this: **0,5** (decimal value).

How can I do that?

Many thanks,

Niklas

**Comment by kevinpapst** [comment, marked as answer]
In code `{{ entry.duration|duration(true) }}`

But you can simply change the duration format in your invoice settings:
<img width="899" alt="Bildschirmfoto 2022-09-01 um 12 32 25" src="https://user-images.githubusercontent.com/533162/187893943-ca79e19d-399b-49d3-bb00-5604695aa0bf.png">

**Comment by ghost** [comment]
Thanks a lot!
